command/init.go

```diff
@@ -104,6 +104,17 @@ job "example" {
 		# Defaults to 1
 		# count = 1
 
+		# Restart Policy - This block defines the restart policy for TaskGroups
+		# attempts defines the number of restarts Nomad will do if Tasks
```

Comment:
We should use `The attempts value ....` here and add a period to the sentence above to avoid confusion